src/components/Header.astro

```diff
@@ -1,76 +1,126 @@
-<div
-  class="grid grid-cols-1 md:grid-cols-2 justify-evenly items-center p-10 bg-dark-blue"
->
-  <div>
-    <div class="mb-6 md:mb-0">
-      <span
-        class="self-center text-5xl font-semibold whitespace-nowrap text-white"
-        ><a
-          href="https://lfx.linuxfoundation.org/tools/mentorship/"
-          class="flex items-center text-center"
-          target="_blank">LFX Mentorship</a
-        ></span
-      >
+<link href="https://fonts.googleapis.com/css2?family=Rajdhani&display=swap" rel="stylesheet">
+<link href="https://fonts.googleapis.com/css2?family=Orbitron:wght@700&display=swap" rel="stylesheet">
```

Comment:
Follow this [guide](https://docs.astro.build/en/guides/fonts/) on how to use custom fonts instead of pulling them from the CDN.